### DISCUSSION TITLE
I can't find my github account in google search

### DISCUSSION THREAD

**Discussion by sajjadul-islam-somon** [community/community#73545] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've tried some ways so that I can find my Github account by searching with my name or username. But my account or any repository isn't showing in the search results even if I search with my Github username. But, if I search by name on Bing it shows up at the top of search results. Please, suggest me what can I do now...

**Comment by ktauchathuranga** [comment]
to my knowledge,

those are different search engines, so you get different results.

**Reply by sajjadul-islam-somon** [reply]
I know it. I just want to clarify that my profile is not private, but still it doesn't show up in google search. Do you have any suggestions?

**Reply by ktauchathuranga** [reply]
mine is showing,
![image](https://github.com/community/community/assets/79928743/b2f2acdc-d981-493f-9676-e3b0bdd8b103)

maybe try to be specific in the search, you may find yours too
🖤
